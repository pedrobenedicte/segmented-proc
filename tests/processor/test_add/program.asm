add r0 <- r1, r2
add r3 <- r4, r5
add r0 <- r1, r2
add r3 <- r4, r5
add r0 <- r1, r2
add r3 <- r4, r5
add r0 <- r1, r2
add r3 <- r4, r5
