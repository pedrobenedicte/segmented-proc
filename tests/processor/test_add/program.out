0a
40
e5
40
0a
40
e5
40
0a
40
e5
40
0a
40
e5
40
